tests/net/net_pkt/src/main.c

```diff
@@ -629,6 +629,49 @@ void test_net_pkt_copy(void)
 		     "Pkt not properly unreferenced");
```

Comment:
for the second commit, please change the subject to `tests: net: pkt: Added a test case for net_pkt_pull()`